plugins/tiddlywiki/markdown/wrapper.js

```diff
@@ -12,78 +12,142 @@ Wraps up the markdown-js parser for use in TiddlyWiki5
 /*global $tw: false */
 "use strict";
 
-var markdown = require("$:/plugins/tiddlywiki/markdown/markdown.js");
+var Remarkable = require('$:/plugins/midnight/remarkable/remarkable.js');
```

Comment:
There are a few coding style things to watch: TW5 uses double quotes for string constants.